web/component/footer.jsx

```diff
@@ -64,13 +75,16 @@ const sections = [
       {
         label: 'Terms of Service',
         link: 'https://www.lbry.com/termsofservice',
+        navigate: '',
       },
       {
         label: 'Privacy Policy',
         link: 'https://lbry.com/privacypolicy',
+        navigate: '',
       },
       {
         label: '2257',
+        link: '',
```

Comment:
Instead of adding a bunch of empty props. You can just add this below

```
let buttonProps = {};
if (link) {
  buttonProps.href = link;
}

if (navigate) {
  buttonProps.navigate = navigate;
}

return <li><Button ... {...buttonProps} ...
```

That should fix the flow errors

Reply:
Something like this?
```
    <footer className="footer">
      {sections.map(({ name, links }) => {
        return (
          <div key={name} className="footer__section">
            <div className="footer__section-title">{name}</div>
            <ul className="ul--no-style">
              {links.map(({ label, link, navigate }) => {
                let buttonProps = {};
                if (link) {
                  buttonProps.href = link;
                }
                if (navigate) {
                  buttonProps.navigate = navigate;
                }

                return (
                  <li key={label}>
                    <Button className="footer__link" {...buttonProps} label={label} />
                  </li>
                );
              })}
            </ul>
          </div>
        );
      })}
    </footer>
```

But flow seems still unhappy:
![image](https://user-images.githubusercontent.com/64950861/83175542-a0789700-a14e-11ea-9115-f35c16bf16a4.png)

Reply:
Ugh. I guess you can just add a flow comment above that line
`// $FlowFixMe`

That should fix it.